### PR TITLE
chore: add E2E tests for duplicate registrations

### PR DESCRIPTION
**Pull request by aberonni**
[AB#33538](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/33538)

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

## Portalicious preview deployment

https://lively-river-04adce503-6552.westeurope.5.azurestaticapps.net

### CODE REVIEW

File: e2e/portalicious/pages/RegistrationBasePage.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    super(page);
+    this.duplicateChip = this.page.getByTestId('duplicate-chip');
+    this.duplicatesBanner = this.page.getByTestId('duplicates-banner');
+  }
+
+  async goToRegistrationPage(
+    page: 'Activity log' | 'Personal information' | 'Debit cards',
+  ) {
+    await this.page.getByRole('tab', { name: page }).click();
+  }
```
Comment (Piotrk39) [inline e2e/portalicious/pages/RegistrationBasePage.ts:20]: I really like this one!

File: e2e/portalicious/pages/RegistrationsPage.ts [hunk 2]
```diff
@@ -505,6 +505,11 @@ class RegistrationsPage extends BasePage {
       validateExactRowCount,
     );
   }
+
+  async assertDuplicateColumnValues(expectedValues: string[]) {
+    const duplicateColumnValues = await this.table.getTextArrayFromColumn(5);
+    expect(duplicateColumnValues).toEqual(expectedValues);
+  }
```
Comment (Piotrk39) [inline e2e/portalicious/pages/RegistrationsPage.ts:512]: I assume in this setup the order of the badges is really important. Is it maybe possible to make this assertion softer. So for example it can contain the specific amount of badges so the order does not matter? Or does it already work like that? Do you think this is possible?

File: e2e/portalicious/tests/ViewAndManagePeopleAffected/ValidateDuplicateBanner.spec.ts [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        name: duplicateRegistrationB.fullName,
+      });
+
+    await expect(duplicateBLink).toBeVisible();
+    await duplicateBLink.click();
+  });
+
+  await test.step('Verify new tab is opened and contains link to orignial duplicate', async () => {
+    await page.waitForTimeout(2000); //waitForNavigation and waitForLoadState do not work in this case
+
```
Comment (Piotrk39) [inline e2e/portalicious/tests/ViewAndManagePeopleAffected/ValidateDuplicateBanner.spec.ts:78]: Yeah only UI elements work in such cases like spinners, loaders etc.

so this is fine

**Review by Piotrk39 — COMMENTED** [review]
It looks really good! Think about doing it full time! Just a minor comment.

File: e2e/portalicious/pages/RegistrationsPage.ts (hunk 2 above)
Comment (aberonni) [reply e2e/portalicious/pages/RegistrationsPage.ts]: Done!

**Comment by Piotrk39** [comment]
It's merged but I saw the change. Good job!
